BlogTemplate/Pages/Edit.cshtml.cs

```diff
@@ -64,16 +68,15 @@ private void UpdatePost(Post newPost, string slug)
         {
             oldPost = _dataStore.GetPost(slug);
             newPost.PubDate = oldPost.PubDate;
+            //newPost.Tags = Request.Form["Tags"][0].Replace(" ", "").Split(",").ToList();

```

Comment:
Whoops, don't know why these commented-out lines keep showing up! I merged in the most recent update of master, which had deleted the commented-out lines of code. Guess I'll need to find a different way to take care of these...